src/coredump/coredump.c

```diff
@@ -919,7 +918,11 @@ static int process_journald_crash(const char *context[], int input_fd) {
         if (r < 0)
                 return r;
 
-        log_info("Detected coredump of the journal daemon itself, diverted to %s.", filename);
+        log_notice("Detected coredump of the journal daemon or PID 1, diverted to %s.", filename);
+
+        log_notice("Due to the special circumstances, coredump collection will now be turned off.");
+        (void) write_string_file("/proc/sys/kernel/core_pattern", "|/bin/false", 0);
```

Comment:
`systemd-journald.service` contains `Restart=always`. So, I think one crash is not a big deal. No?